### DISCUSSION TITLE
posts/save-scene-performance-tuning/

### DISCUSSION THREAD

**Discussion by giscus** [LeeSM0518/LeeSM0518.github.io#3] [Announcements]
# posts/save-scene-performance-tuning/

데이터 동기화 방식과 배치 주기를 변경하여 저장 시간을 20분에서 1초로 개선

https://leesm0518.github.io/posts/save-scene-performance-tuning/

**Comment by hongbeomi** [comment]
1200배... 대단하네요~

**Reply by LeeSM0518** [reply]
감사합니다😊
